library/gradle/dokka-config.gradle

```diff
@@ -1,4 +1,4 @@
-apply plugin: libs.dokka.gradlePluginId
+apply plugin: libs.plugins.dokka.get().pluginId
```

Comment:
yes, but only if we are not using the same dependency in classpath, otherwise gradle is not recognizing version of same dependency in classpath and raises incompatibility issue. Although I might just not know enough about it.

Reply:
I think this is the issue: https://github.com/gradle/gradle/issues/18236
It is closed but it was automatically resolved, not fixed.